zio-cli/shared/src/main/scala/zio/cli/AutoCorrect.scala

```diff
@@ -0,0 +1,33 @@
+package zio.cli
+
+object AutoCorrect {
```

Comment:
Maybe make this package private to `cli` by adding `private [cli] object AutoCorrect`. Otherwise it will appear in the Scaladoc and users will wonder when to use it.